python/ray/experimental/state.py

```diff
@@ -649,14 +660,16 @@ def micros_rel(ts):
                 if parent_info:
                     parent_worker = workers[parent_info["worker_id"]]
                     parent_times = self._get_times(parent_info)
+                    parent_profile = task_info.get(
+                        task_table[task_id]["TaskSpec"]["ParentTaskID"])
                     parent = {
                         "cat": "submit_task",
                         "pid": "Node " + parent_worker["node_ip_address"],
                         "tid": parent_info["worker_id"],
-                        "ts": micros_rel(task_profiles[task_table[task_id]
-                                                       ["TaskSpec"]
-                                                       ["ParentTaskID"]]
-                                          ["get_arguments_start"]),
+                        "ts": micros_rel(
+                            parent_profile and
```

Comment:
With truncation it's possible we don't have some parent profiles. We are just ignoring them for now.